src/CMakeLists.txt

```diff
@@ -222,6 +222,7 @@ endif()
 
 find_package(spdlog REQUIRED)
 message(STATUS "spdlog: ${spdlog_VERSION}")
+add_definitions(-DFMT_DEPRECATED_OSTREAM=1)
```

Comment:
I think it would be more modern in utl/CMakeLists.txt to use

target_compile_definitions(utl PUBLIC FMT_DEPRECATED_OSTREAM=1)

All users of utl should then get the define.

Reply:
https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#dont-use-macros-that-affect-all-targets-in-a-directory-tree-like-include_directories-add_definitions-or-link_libraries

Reply:
https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#declare-compile-definitions-with-target_compile_definitions

Reply:
Thanks, I don't deal with cmake enough to know what's best.